Documentation/README.md

```diff
@@ -48,3 +48,11 @@ You can build the .NET Core SDK and generate a source tarball in a specific dire
 ```console
 ./build-source-tarball.sh <path-to-tarball-root>
 ```
+
+After this completes, you can take the output tarball and build .NET Core again without an internet connection.
+
+```console
+tar -xf <path-to-tarball-root> -C <path-to-extraction-root>
```

Comment:
The output isn't a tarball, so no extract is required.  The path-to-tarball-root is the directory to put everything so a tarball could be created from it, but the tarball is not created.  You can change this to:
```
cd <path-to-tarball-root>
./build.sh
```